<launch>

  <arg name="node_name" default="position_driver_node" />
  <arg name="sub_topic" default="cmd_pos" />
  <arg name="pub_topic" default="odom" />
  <arg name="tolerance" default="0.1" />

  <arg name="cw_ccw" />
  <arg name="start_top" />
  <arg name="interrupt" />

  <arg name="reduction_ratio" />
  <arg name="wheel_radius" />
  <arg name="pulse_per_spin" />

  <node name="$(arg node_name)" pkg="raspberry_pi_driver" type="raspbery_pi_driver_node">
    <remap from="sub_topic" to="$(arg sub_topic)" />
    <remap from="pub_topic" to="$(arg pub_topic)" />

    <param name="tolerance"  value="$(arg tolerance)" />
    <param name="cw_ccw"     value="$(arg cw_ccw)" />
    <param name="start_stop" value="$(arg start_stop)" />
    <param name="interruput" value="$(arg interruput)" />

    <param name="reduction_ratio" value="$(arg reduction_ratio)" />
    <param name="wheel_radius"    value="$(arg wheel_radius)" />
    <param name="pulse_per_spin"    value="$(arg pulse_per_spin)" />
  </node>

</launch>
